src/dnx/dnx.cpp

```diff
@@ -235,7 +235,11 @@ int CallApplicationProcessMain(int argc, LPTSTR argv[])
     bool fSuccess = true;
     HMODULE m_hHostModule = nullptr;
 #if CORECLR_WIN
-    LPCTSTR pwzHostModuleName = _T("dnx.coreclr.dll");
+#if ONECORE
+        LPCTSTR pwzHostModuleName = _T("dnx_onecore.coreclr.dll");
```

Comment:
@davidfowl I infer that in this case we also want to have dnx.onecore.dll and dnx.win32.dll instead of dnx_onecore.dll and dnx_win32.dll (the [other](https://github.com/aspnet/dnx/pull/1886/files#diff-4d19362f790908c080d93fa6c5e708adR13) PR)